packages/editor/src/components/panel-color/index.js

```diff
@@ -17,6 +18,12 @@ import withColorContext from '../color-palette/with-color-context';
 import { getColorObjectByColorValue } from '../colors';
 
 function PanelColor( { colors, title, colorValue, initialOpen, ...props } ) {
+	deprecated( 'wp.editor.PanelColor', {
+		version: '4.2',
+		alternative: 'wp.editor.PanelColorSettings',
```

Comment:
Unfortunately wp.editor.PanelColorSettings is not a viable alternative to wp.components.PanelColor. wp.components.PanelColor allows a block to specify the colors that appear instead of using the default or theme specified ones which is useful for some blocks that only make sense with some colors or blocks that should expand the color options, e.g., a drawing block.

Reply:
Right, I noted that we could have an alternate option in the PR:

> One note is that deprecating `wp.components.PanelColor` will mean there's no component to provide a color panel that isn't wrapped inside `withColorContext`. As such, it may be desired to deprecate `wp.editor.PanelColor`, while _keeping_ `wp.components.PanelColor` and simply fixing the visual bug noted in #10367.

Happy to tackle that in this PR too, if you'd like @jorgefilipecosta :)

Reply:
Also just to clarify, it makes sense to me to keep the version in `components` and get rid of the one in `editor` but is there any reason to do the reverse instead?

Reply:
This will be fixed by #10457!